demo/config/tools/handbook_creator/templates/command_packets.html.erb

```diff
@@ -4,6 +4,8 @@
     <h2><%= "#{packet.target_name} #{packet.packet_name}" %></h2>
     <div><span style="font-size:18px;background:#555555;color:white;"><%= packet.hazardous ? 'Hazardous' : '' %></span></div>
     <div><%= packet.description %></div>
+    <% items = packet.sorted_items.reject{|item| ignored[packet.target_name].include? item.name} %>
+    <% next if items.empty? %>
```

Comment:
I don't really like all this logic here but I needed to create the item list so I could skip the parameter table if we didn't have any parameters (they were all ignored).